configuration_manager/config_logger.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements; and to You under the Apache License,
+#  Version 2.0."
+
+import os
+import logging
+import logging.config
+from functools import reduce
+import operator
+
+
+class ConfigLogger(object):
+    """Creates loggers."""
+    @classmethod
+    def config_basic_settings(cls, name, target_directory):
+        exit(1)  # TODO: add fallback configuration settings
```

Comment:
Throw exception maybe? Or a msg what is wrong. 
This is a hard exit without any details why